### ISSUE TITLE
MediaRecorder.isTypeSupported example includes incorrect codec label

### ISSUE THREAD

**Issue by jamiebuilds** [mdn/content#36567]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/MediaRecorder/isTypeSupported_static

### What specific section or headline is this issue about?

Examples

### What information was incorrect, unhelpful, or incomplete?

In `MediaRecorder.isTypeSupported`, in the examples section, in the `types` array:

```js
const types = [
  "video/webm",
  "audio/webm",
  "video/webm;codecs=vp8",
  "video/webm;codecs=daala",
  "video/webm;codecs=h264", // << This is not a valid codecs=<label>
  "audio/webm;codecs=opus",
  "video/mp4",
];
```

Link: https://developer.mozilla.org/en-US/docs/Web/API/MediaRecorder/isTypeSupported_static#examples

`codecs=h264` is not the label used for H.264 aka AVC1. It should be `codecs=avc1` or one of the many more [specific profiles](https://cconcolato.github.io/media-mime-support/#avc_codecs)

### What did you expect to see?

```js
const types = [
  "video/webm",
  "audio/webm",
  "video/webm;codecs=vp8",
  "video/webm;codecs=daala",
  "video/webm;codecs=avc1", // << fixed
  "audio/webm;codecs=opus",
  "video/mp4",
];
```

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/mediarecorder/istypesupported_static`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/MediaRecorder/isTypeSupported_static
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/mediarecorder/istypesupported_static/index.md
* Last commit: https://github.com/mdn/content/commit/f0e465f756df374b51e36668f5cc2559c26b6767
* Document last modified: 2024-04-05T01:16:49.000Z

</details>

**Comment by skyclouds2001** [comment]
chrome says it is supported

![image](https://github.com/user-attachments/assets/0e9a6b37-b50a-44f8-8057-f24bdb288eb3)

**Comment by Josh-Cena** [comment]
Yeah I just checked. Apparently `h264` is as "non-standard" as `avc1` (because theoretically WebM only allows royalty-free codecs), because neither is listed here: https://wiki.whatwg.org/wiki/video_type_parameters#WebM In reality browsers treat them as equivalent for `video/webm` anyway, and I see no signs of one being preferred over the other in browsers' source code, so I think this example is fine as-is. Unless there are official guidelines to say prefer one to the other.
